scripts/ci

```diff
@@ -55,11 +60,21 @@ function build() {
 }
 
 function ci() {
-    build $1
+    build $1 $2
     command_tests
 }
 
-if [ "${CI_CHECK_FORMAT}" == "1" ] ; then
+build_type=$1
+full_build=$2
+if [ "" == "${full_build}" ] ; then
```

Comment:
`"" == "${full_build}"` -> `"${full_build}" == ""`

Reply:
this applies to the conditionals below too

Reply:
Changed the order